.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 lib
 .eslintcache
 package-lock.json
+yarn.lock
```

Comment:
I added this line (not sure but it seems legit if we don't want to version the file)